src/components/view-3d/water.tsx

```diff
@@ -7,10 +7,18 @@ import { mToViewUnit, PLANE_WIDTH, planeHeight } from "./helpers";
 import { observer } from "mobx-react-lite";
 import { useStores } from "../../use-stores";
 import { useUpdate } from "react-three-fiber";
+// Very simple shaders. They won't work great in 3D view, as there's no lightning there. But 3D view is used only
```

Comment:
```suggestion
// Very simple shaders. They won't work great in 3D view, as there's no lighting there. But 3D view is used only
```
Unless there's a thunderstorm simulation mode I'm not aware of. 😉

Reply:
Thanks ;)